index.js

```diff
@@ -564,6 +580,10 @@ function trimLastSlash (path) {
   return path
 }
 
+function removeDuplicateSlashes (path) {
+  return path.replace(/\/\/+/g, '/')
```

Comment:
It's not a good idea to use regex matching in a find method. It's a hot path. You can run benchmarks to see the performance drop.